HLTrigger/special/src/HLTRegionalEcalResonanceFilter.cc

```diff
@@ -1076,23 +1119,23 @@ bool HLTRegionalEcalResonanceFilter::checkStatusOfEcalRecHit(const EcalChannelSt
   return true; 
 }
 
+// use function in DataFormats/Math/interface/deltaPhi.h
 
-float HLTRegionalEcalResonanceFilter::DeltaPhi(float phi1, float phi2){
-
-  float diff = fabs(phi2 - phi1);
+// float HLTRegionalEcalResonanceFilter::DeltaPhi(float phi1, float phi2){
   
-  while (diff >acos(-1)) diff -= 2*acos(-1);
-  while (diff <= -acos(-1)) diff += 2*acos(-1);
-  
-  return diff; 
+//   float diff = phi2 - phi1;
+//   while (diff > M_PI) diff -= 2.*M_PI;
+//   while (diff <= -M_PI) diff += 2.*M_PI;
 
-}
+//   return diff; 
+
+// }
 
 
 float HLTRegionalEcalResonanceFilter::GetDeltaR(float eta1, float eta2, float phi1, float phi2){
   
   return sqrt( (eta1-eta2)*(eta1-eta2) 
-	       + DeltaPhi(phi1, phi2)*DeltaPhi(phi1, phi2) );
+	       + deltaPhi(phi1, phi2)*deltaPhi(phi1, phi2) );
```

Comment:
I would suggest to use `std::hypot(eta1-eta2, deltaPhi(phi1, phi2))`, unless there already is some "delta R" function we can reuse.